packages/web-pkg/src/components/AppTemplates/AppWrapper.vue

```diff
@@ -6,6 +6,7 @@
       :main-actions="fileActions"
       :drop-down-menu-sections="dropDownMenuSections"
       :drop-down-action-options="dropDownActionOptions"
+      :is-auto-save-enabled="isAutoSaveEnabled"
```

Comment:
slight wording thingy: what about `hasAutoSave` instead of `isAutoSaveEnabled`? 🤔